resources/twitchcommands.py

```diff
@@ -618,6 +618,48 @@ def gamecontrol():
         #if (PREFIX + "setecotime" == str(args[0]).lower() or PREFIX + "ecotime" == str(args[0]).lower()) and len(args) >= 2:
         #    sendForm("(set! (-> *FACT-bank* eco-full-timeout) (seconds " + str(args[1]) + "))")
         #    message = ""
+
+        if PREFIX + "big-head" == str(args[0]).lower() and on_check("big-head") and cd_check("big-head"):
+            active_check("big-head",
+            "(begin (logior! (-> *pc-settings* cheats) (pc-cheats big-head)) (logclear! (-> *pc-settings* cheats-known) (pc-cheats big-head)))",
+            "(logclear! (-> *pc-settings* cheats) (pc-cheats big-head))")
+            message = ""
+
+        if PREFIX + "small-head" == str(args[0]).lower() and on_check("small-head") and cd_check("small-head"):
+            active_check("small-head",
+            "(begin (logior! (-> *pc-settings* cheats) (pc-cheats small-head)) (logclear! (-> *pc-settings* cheats-known) (pc-cheats small-head)))",
+            "(logclear! (-> *pc-settings* cheats) (pc-cheats small-head))")
+            message = ""
+
+        if PREFIX + "big-fist" == str(args[0]).lower() and on_check("big-fist") and cd_check("big-fist"):
+            active_check("big-fist",
+            "(begin (logior! (-> *pc-settings* cheats) (pc-cheats big-fist)) (logclear! (-> *pc-settings* cheats-known) (pc-cheats big-fist)))",
+            "(logclear! (-> *pc-settings* cheats) (pc-cheats big-fist))")
+            message = ""
+
+        if PREFIX + "big-head-npc" == str(args[0]).lower() and on_check("big-head-npc") and cd_check("big-head-npc"):
+            active_check("big-head-npc",
+            "(begin (logior! (-> *pc-settings* cheats) (pc-cheats big-head-npc)) (logclear! (-> *pc-settings* cheats-known) (pc-cheats big-head-npc)))",
+            "(logclear! (-> *pc-settings* cheats) (pc-cheats big-head-npc))")
+            message = ""
+
+        if PREFIX + "huge-head" == str(args[0]).lower() and on_check("huge-head") and cd_check("huge-head"):
+            active_check("huge-head",
+            "(begin (logior! (-> *pc-settings* cheats) (pc-cheats huge-head)) (logclear! (-> *pc-settings* cheats-known) (pc-cheats huge-head)))",
+            "(logclear! (-> *pc-settings* cheats) (pc-cheats huge-head))")
+            message = ""
+
+        if PREFIX + "mirror" == str(args[0]).lower() and on_check("mirror") and cd_check("mirror"):
+            active_check("mirror",
+            "(begin (logior! (-> *pc-settings* cheats) (pc-cheats mirror)) (logclear! (-> *pc-settings* cheats-known) (pc-cheats mirror)))",
```

Comment:
this turns on the cheat, and also hides it from the cheat menu so player can't go and turn it off 😈